content/docs/5.1/reference/architecture/overview.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Overview
+description: Core concepts and architecture behind TiKV.
+menu:
+    "5.1":
+        parent: Architecture
+        weight: 1
+---
+
+This page discusses the core concepts and architecture behind TiKV, including:
+
+* The [APIs](#apis) that applications can use to interact with TiKV
+* The basic [system architecture](#system) underlying TiKV
+* The anatomy of each [instance](#instance) in a TiKV installation
+* The role of core system components, including the [Placement Driver](#placement-driver), [Store](#store), [Region](#region), and [Node](#node)
```

Comment:
Do you mean `the terminology of the TiKV cluster`

Reply:
Since, region is not a "role", it's a terminology of our own.

Reply:
yes

Reply:
fixed